ppci/lang/c/preprocessor.py

```diff
@@ -864,6 +866,7 @@ def handle_define_directive(self, directive_token):
 
         # Handle function like macros:
         variadic = False
+        named_variadic = False
```

Comment:
Interesting feature again of C!

Idea for a simplification:
Set `variadic` to `None` here. Later, set `variadic` to either `__VA_ARGS__` or to the given id. That saves the extra `named_variadic` variable.